.github/workflows/ci.yml

```diff
@@ -32,14 +32,8 @@ jobs:
             - name: Run prettier
               run: yarn testPrettierAll
 
-            - name: Run tsc build
-              run: yarn buildTsc
-
-            - name: Run tsc buildTests
-              run: yarn lerna run buildTests
```

Comment:
Added side effect: The `istJustJavascript` and `dist/tests` dirs are pretty much things of the past.
I haven't entirely gone rid of them at this point, but we're pretty close now!

Reply:
Yes - this is certainly not something I want to change in this PR :)

Was just thinking that jest was one of the main consumers of `itsJustJavascript`, and that's now gone.